docs/index.md

```diff
@@ -5,46 +5,33 @@ hide:
-comes with built-in components that make it easy to organize and control how data is accessed and 
-managed. This feature, called "Scenario Management," requires very little setup in Python.
+At the same time, Taipy is designed for creating data-driven back-end applications. It comes with built-in components that make it easy to organize and control how data is accessed and managed. Pipeline orchestration is very easy to implement through a graphical editor (Taipy Studio) and an intuitive Python API. 
 
-With Taipy Studio, a graphical configuration editor, setting up your scenarios (which include data 
-flows and pipelines) has become much simpler. This tool offers a user-friendly interface where you 
-can easily drag and drop elements, customize layouts, and integrate features effortlessly.
+The capability to:
 
-Taipy is a versatile tool that enables both individuals and businesses to create applications 
-without needing advanced coding skills. One of its key benefits is how it speeds up development for 
-both the front-end and back-end aspects, covering everything from initial prototypes to fully 
-scalable, ready-to-use applications.
+- Register pipeline single execution. 
+- Perform what-if analysis (comparing different executions/runs). 
+- Explore your past executions, comparing them.
+
+Is provided by Taipy’s “Scenario Management”, one of Taipy’s central concepts.
 
 **Front-End Functionalities:**
 
-- Creating a user interface requires a solid grasp of Python programming fundamentals.
-- Taipy is purposefully engineered to prioritize user-friendliness, resulting in a straightforward 
-  and intuitive process of user interface creation.
-- No prior knowledge of web design is necessary, and it eradicates the need for any prerequisites 
-  related to CSS and HTML.
-- Leveraging augmented Markdown syntax, Taipy GUI aids users in seamlessly generating their desired
-  web pages.
+- Taipy provides all the GUI components needed to build interactive front-ends.
+- No prior knowledge of web design is necessary, and any prerequisites related to CSS and HTML.
+- Leveraging augmented Markdown syntax, Taipy GUI aids users in seamlessly generating their desired web pages.
+- Taipy’s ability to really customize your GUI in terms of layout, graphical events, Style/CSS, etc is one of Taipy’s strongest points.
 
 **Back-End Functionalities:**
 
-- Taipy provides extensive support for establishing resilient pipelines capable of managing diverse
-  scenarios.
+- Taipy provides extensive support for orchestrating resilient pipelines capable of managing diverse situations.
 - The process of modeling Directed Acyclic Graphs (DAGs) is simplified through Taipy Studio.
-- Enhancing the overall performance of Taipy applications, data caching, parallel executions of 
-  tasks, data source scoping are all integrated.
+- Enhancing the overall performance of Taipy applications, task caching, parallel executions of tasks, data source scoping are all integrated.
```

Comment:
"Enhancing the overall performance of Taipy applications". What does performance mean?

In a technical document, the performance usually means a short computation time or a low memory usage... I guess here we want to advertise that it makes the developer and the end user efficient in their work. This is really not clear to me.

Reply:
Correct, I have updated it to: "Improving the overall effectiveness of Taipy applications."